package.json

```diff
@@ -31,7 +31,7 @@
     "dotenv": "^6.1.0",
     "mocha": "^5.2.0",
     "mocha-junit-reporter": "^1.18.0",
-    "nyc": "^12.0.2",
+    "nyc": "^13.3.0",
```

Comment:
You need to run `npm install` after this change and check in the updated `package-lock.json` as well to fix the build failure